.circleci/config.yml

```diff
@@ -1,11 +1,95 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@0.2.0
+  architect: giantswarm/architect@0.4.2
+
+e2eTest: &e2eTest
+  machine: true
+  working_directory: /home/circleci/.go_workspace/src/github.com/giantswarm/cluster-autoscaler-app
+  steps:
+    - checkout
+
+    - attach_workspace:
+        at: .
+
+    - run:
+        name: Get kubectl binary
+        command: |
+          curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+          chmod +x ./kubectl
+
+    - run:
+        name: Get e2ectl binary
+        command: |
+          wget -q $(curl -sS https://api.github.com/repos/giantswarm/e2ectl/releases/latest | grep browser_download_url | head -n 1 | cut -d '"' -f 4)
+          chmod +x ./e2ectl
+    - run:
+        name: Init cluster
+        command: |
+          ./e2ectl cluster create
+          cp $(./e2ectl kubeconfig path) ${E2E_TEST_DIR}/kubeconfig
+
+    - run:
+        name: labeling master node with kubernetes.io/role as same as production
+        command: |
+          export KUBECONFIG=$(./e2ectl kubeconfig path)
+          ./kubectl label --all nodes kubernetes.io/role=master
```

Comment:
labelling master node with this since kind using kubeadm which is using different lable.